requirements.txt

```diff
@@ -197,17 +201,17 @@ python-dateutil==2.8.2
     #   botocore
 python-json-logger==2.0.2
     # via notifications-utils
-pytz==2021.3
+pytz==2022.1
     # via
     #   celery
     #   notifications-utils
 pyyaml==5.4.1
     # via
     #   awscli
     #   notifications-utils
-redis==3.5.3
+redis==4.3.1
```

Comment:
there's a few breaking changes wrapped up in here. i had a quick skim through and it appears okay but it would be good to test this, especially with gunicorn since we had the monkeypatching stuff i just wanna be sure we're handling correctly

Reply:
We're running this version of redis in admin, which seems to do the same monkeypatching. Let me know if you think there is more that we should be testing apart from checking everything runs on preview - if so, I'll do this in it's own PR to be safe.

Reply:
ah had missed that it was already onadmin. yeah thats fine then.